frontend/init/turbo.js

```diff
@@ -0,0 +1,30 @@
+import * as Turbo from '@hotwired/turbo';
+
+Turbo.setProgressBarDelay(100);
+
+const PRESERVE_SCROLL_SELECTOR = '[data-turbo-preserve-scroll-position]';
+let scrollPositions = {};
+
+const storeScrollPositions = () => {
+  scrollPositions = {};
+  const containers = document.querySelectorAll(PRESERVE_SCROLL_SELECTOR);
+
+  containers.forEach(container => {
+    scrollPositions[container.id] = container.scrollTop;
+  });
+};
+
+const restoreScrollPositions = () => {
+  const containers = document.querySelectorAll(PRESERVE_SCROLL_SELECTOR);
+
+  containers.forEach(container => {
+    const scrollPosition = scrollPositions[container.id];
+
+    if (scrollPosition && scrollPosition !== container.scrollTop) {
+      container.scrollTop = scrollPosition;
+    }
+  });
+};
+
```

Comment:
Can we just use `id`s and `#` fragment identifier, please?

Reply:
In Firefox and Chromium, this works fine with navigating back and forth.

Reply:
As explained, unfortunately, this won’t prevent the scroll position from changing while navigating between different contributor profiles.